workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/androidx/WorkflowLifecycleOwner.kt

```diff
@@ -225,6 +225,9 @@ internal class RealWorkflowLifecycleOwner(
     }
 
     localLifecycle.currentState = when {
+      event != null -> {
+        event.targetState
```

Comment:
Don't we need to make sure the local lifecycle isn't greater than the parent? E.g. if the parent hasn't STARTED yet, we shouldn't either.

Reply:
I think this should be renamed to `parentEvent`  since the events should originate from the parent lifecycle. For example, onStateChanged ensures that the local lifecycle mirrors the parent's state and doesn't exceed it.